.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/devcontainers/typescript-node:1-22
+
+RUN apt-get update \
+    && apt-get install -y curl --no-install-recommends \
+    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL https://raw.githubusercontent.com/devcontainers/features/main/src/docker-in-docker/install.sh | bash \
+    && rm -f install.sh \
+    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
```

Comment:
refs:
- https://github.com/devcontainers/features/blob/main/src/docker-in-docker/install.sh
- https://scrapbox.io/shimizukawa/devcontainer%E3%81%A7Docker_in_Docker%E7%92%B0%E5%A2%83%E3%82%92%E6%A7%8B%E7%AF%89%E3%81%97Codespaces%E3%81%8B%E3%82%89https%E8%BB%A2%E9%80%81%E3%81%99%E3%82%8B